python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -275,6 +275,13 @@ def _output_ready_callback(task_index, output: RefBundle):
         def _task_done_callback(task_index, inputs):
             # We should only destroy the input bundle when the whole task is done.
             # Otherwise, if the task crashes in the middle, we can't rerun it.
+            refs = [input[0] for input in inputs.blocks]
```

Comment:
Out of scope for this PR, but `input[0]` was confusing here. Didn't fully understand it until I read the `RefBundle` definition.

https://github.com/ray-project/ray/blob/fd9a02e9cef9cff0e58e99274622c651e1227f4c/python/ray/data/_internal/execution/interfaces/ref_bundle.py#L31